Directory.Build.props

```diff
@@ -1,11 +1,12 @@
+<?xml version="1.0"?>
 <Project>
   <PropertyGroup>
-    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
`VersionPrefix` is necessary to create preview packages for NuGet

Reply:
I'd suggest that we just write the generated preview version to the `version.txt` with a small script.